pkg/virt-launcher/virtwrap/api/schema.go

```diff
@@ -928,6 +928,7 @@ type ChannelSource struct {
 //BEGIN Sound -------------------
 
 type SoundCard struct {
+	Alias *Alias `xml:"alias,omitempty"`
```

Comment:
I am curious why do we make this "optional"?

Reply:
it isn't a must from libvirt pov. The related field at `staging/src/kubevirt.io/api/core/v1/schema.go` is not optional. That's the reasoning that I used but let me know if I should change that (and perhaps other places that might be wrong too)

Reply:
It does not really make a difference imo.

Reply:
I think I see now that we are not verifying for all devices right now (like .e.g tablet, gpu, ...) if we have name duplicates. I htink that deserves another bug. We can then ensure that no duplicates exist also for sound.

Reply:
I just thought I am missing something.

Reply:
@rmohr I'll address duplicates for other devices too in another PR plus this https://github.com/kubevirt/kubevirt/pull/6829#discussion_r756148440

Reply:
Nice. Thanks a lot :)